core/config/envvar/envvar.go

```diff
@@ -0,0 +1,92 @@
+package envvar
+
+import (
+	"fmt"
+	"log"
+	"os"
+
+	"github.com/pkg/errors"
+	"go.uber.org/zap/zapcore"
+
+	"github.com/smartcontractkit/chainlink/core/config/parse"
+)
+
+var (
+	LogLevel    = New("LogLevel", parse.LogLevel)
+	RootDir     = New("RootDir", parse.HomeDir)
+	JSONConsole = New("JSONConsole", parse.Bool)
+	LogToDisk   = New("LogToDisk", parse.Bool)
+	LogUnixTS   = New("LogUnixTS", parse.Bool)
```

Comment:
Only the log vars needed to move out of config to be accessible, but it could still be beneficial to move all the vars since this pattern has the advantage of doing the reflection up front just once, instead of lazily each time. Although, it may be better to revisit after 1.18 with generics, which will simplify a lot of this code.